src/styles/blocks/_button.scss

```
@@ -0,0 +1,20 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 56px;
+  text-decoration: none;
+  color: $color-white;
+  font-weight: 700;
+  background-color: $main-accent-color-red;
+  transition: all 0.3s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
just list properties which will change
Also, create variable for transitionDuraiton
```suggestion
  transition: background-color $transitionDuraiton;
```